extension.js

```diff
@@ -33,6 +33,12 @@ function activate(context) {
 			],
 			"url": "https://raw.githubusercontent.com/letzya/tyk-schemas/main/schema_apidef_lean.json"
 		},
+		{
+			"fileMatch": [
+				"oasapidef.*.json"
```

Comment:
do you also want to add this match `"TykOasApiDef-*.json"`? and for the classic def ` "TykDefinition-*.json"`?

Reply:
yeh, as another option. thanks